ignition/src/GazeboWrapper.cpp

```diff
@@ -384,6 +384,19 @@ bool GazeboWrapper::initialized()
     }
 }
 
+double GazeboWrapper::getSimulatedTime() const
+{
+    auto dt = pImpl->gazebo.physics.maxStepSize;
+    auto iterationCount = pImpl->gazebo.server->IterationCount();
+
+    if (!iterationCount || !pImpl->gazebo.server) {
+        gymppError << "Failed to get the simulated time" << std::endl;
+        return 0.0;
+    }
+
+    return dt * iterationCount.value();
```

Comment:
This is not related to the PR as it is a limitation of the Gazebo Server API, but these seems error prone, as it would break if at some point Gazebo changes the step size during a simulation (just wanted to comment this, even if nothing can be changed in this PR).

Reply:
Good point, thanks! I think we can just leave a comment in the code as a reminder